lib/rabbit_feed/event_routing.rb

```diff
@@ -86,8 +89,8 @@ def accepted_routes
     end
 
     def handle_event event
-      application = find_application event.application
-      raise RoutingError.new "No routing defined for application with name: #{event.application}" unless application.present?
+      application = find_application event
+      raise RoutingError.new "No routing defined for application with name: #{event.application} for events named: #{event.name}" unless application.present?
```

Comment:
very minor... should this define that it was jt found in the default "application" either ?